bsip-0056.md

```diff
@@ -0,0 +1,314 @@
+
+## Annulment of vesting balances
+
+This new type of vesting balance must be able to know when it was `last used` (created, deposited, withdrawn, settled). This 
+timestamp is updated every time the vesting balance is used. A new committee parameter is introduced: `cleanup_unused_vesting_balances_after_seconds` (seconds). If the vesting balance was unused for longer than `cleanup_unused_vesting_balances`, it is annuled during maintenance interval. This applies only to managed vesting balances.
+
+Additional checks during maintenance:
+ - Query all unused managed vesting balances
+   - If it has a balance greater than 0, liquidate by forcing a withdrawal (just as if the user would manually initiate a withdrawal), no fee payment necessary. This action may or may not update the `last used` time (developers choice to what is feasible). Deletion of this object will not happen within this maintenance interval
+   - If the balance is 0, delete the vesting balance object, no return of any fees
+
+## Honoring White and Blacklisting
+
+Managed vesting balances must account for white- and blacklisted assets and accounts, and behave in that manner as if the asset would be held directly by the account.
+
+Vesting Balances can only be created or charged (deposit) if
+- the owner and the fee paying account could also directly hold the contained asset (black- and whitelist check)
+- the owner of the vesting balance has not blacklisted (or whitelisted) the fee paying account and/or the manager
+
+Vesting Balances can only be settled if 
+- the owner and the fee paying account also directly hold the contained asset (black- and whitelist check)
+- account blacklisting or whitelisting is not checked to disallow abuse. It is considered sufficient to check on creation
+
+Vesting Balances can be withdrawn from if 
+- the owner could also directly hold the contained asset (black- and whitelist check)
+
+Note:
+- This BSIP is meant to honor the black- and whitelist for any vesting balance, independent of its policy (i.e. also the already existing vesting balances!)
+
+# Economy
+
+It is expected that many managed vesting balances are to be created to realize the connected CEX. 
+This requires that the creation and usage fees should be small, as a trade-off managed vesting balances annul themselves if unused to not take up RAM.
+
+Initial fees (proposal):
+ - creation fee: similar to half the vesting balance create fee
+ - deposit and withdrawal fee: similar to a transfer operation fee
+ - settlement fee: limit_order fee * amount of contained `deltas`
+
+# Examples
+
+## Example: What amount should be allowed to be settled?**
+
+Assume that a user creates a vesting balance A with amount 100 on it (asset does not matter). Now he initiates 
+`vesting_balance_withdrawal_operation` with `amount = 60`, `delay_seconds = 100`. At this point following holds:
+ - the vesting balance *manager* can settle, increasing is possible, reduction from from 100 to 0 as well
```

Comment:
Confusing wording.  Maybe "the vesting balance manager can settle resulting in a possible increase in balance or possible decrease in balance from 100 to 0".  If acceptable, I suggest a similar change to "the vesting balance manager can settle, increasing is possible, only reduction from 40 to 0"

Reply:
Refined